host.py

```diff
@@ -0,0 +1,49 @@
+from node import Node
+
+class Host(Node):
+    """A class that will represent hosts.
+
+    Args:
+        ns: An instance of the NetworkSimulator class.
+        node_id: The network address of the host.
+        link: The link that the host is connected to.
+        flows: All the flows that are going out of the host.
+
+    Attributes:
```

Comment:
For attributes, put the type of the attribute afterwards, like:

ns (NetworkSimulator): ...
node_id (int): ...